keylime/ima.py

```diff
@@ -56,106 +56,116 @@
-                   'ima-sig':'d-ng|n-ng|sig;',
-                   }
 
 def ima_eventname_parse():
     pass
 
+
 def ima_eventdigest_ng_parse():
     pass
 
 
 def ima_eventname_ng_parse():
     pass
 
+
 def ima_eventsig_parse():
     pass
 
-supported_fields ={
-                   "n":ima_eventname_parse,
-                   "d-ng":ima_eventdigest_ng_parse,
-                   "n-ng":ima_eventname_ng_parse,
-                   "sig":ima_eventsig_parse,
-                   }
 
-def read_measreument_list_bin(path,whitelist):
+supported_fields = {
+    "n": ima_eventname_parse,
+    "d-ng": ima_eventdigest_ng_parse,
+    "n-ng": ima_eventname_ng_parse,
+    "sig": ima_eventsig_parse,
+}
+
+
+def read_measreument_list_bin(path, whitelist):
     raise Exception("not implementated fully yet")
     f = open(path, 'rb')
 
     while True:
-        template={}
-        (template['pcr'],template['digest'],template['name_len']) = read_unpack(f,"<I20sI")
+        template = {}
+        (template['pcr'], template['digest'],
+         template['name_len']) = read_unpack(f, "<I20sI")
 
-        if template['name_len']>TCG_EVENT_NAME_LEN_MAX:
-            raise Exception("Error event name too long %d",template['name_len'])
+        if template['name_len'] > TCG_EVENT_NAME_LEN_MAX:
+            raise Exception("Error event name too long %d",
+                            template['name_len'])
 
-        name = read_unpack(f,"<%ds"%template['name_len'])[0]
+        name = read_unpack(f, "<%ds" % template['name_len'])[0]
 
-        is_ima_template = name=='ima'
+        is_ima_template = name == 'ima'
         if not is_ima_template:
-            template['data_len']=read_unpack(f,"<I")[0]
-            print("reading ima len %d"%template['data_len'])
+            template['data_len'] = read_unpack(f, "<I")[0]
+            print("reading ima len %d" % template['data_len'])
         else:
             template['data_len'] = SHA_DIGEST_LEN+TCG_EVENT_NAME_LEN_MAX+1
 
-        template['data'] = read_unpack(f,"<%ds"%template['data_len'])[0]
+        template['data'] = read_unpack(f, "<%ds" % template['data_len'])[0]
 
         if is_ima_template:
-            field_len = read_unpack(f,"<I")[0]
-            extra_data=read_unpack(f,"<%ds"%field_len)[0]
-            template['data']+= extra_data
+            field_len = read_unpack(f, "<I")[0]
+            extra_data = read_unpack(f, "<%ds" % field_len)[0]
+            template['data'] += extra_data
 
-        print("record %s"%template)
+        print("record %s" % template)
 
         if template['name'] not in list(defined_templates.keys()):
-            template['name']='ima'
-        template['desc-fmt']=defined_templates[template['name']]
+            template['name'] = 'ima'
+        template['desc-fmt'] = defined_templates[template['name']]
 
         #tokens = template['desc-fmt'].split('|')
 
-        import pdb; pdb.set_trace()
+        import pdb
+        pdb.set_trace()
```

Comment:
Why we need debugger here?

Reply:
Part of the old code reformatted by my IDE's linter... @jetwhiz , do we need pbd still? I imagine not.

Reply:
Yeah I expect this was just used for debugging the loading of the binary IMA list.

Reply:
Ok, @lukehinds. Btw, I haven't started the code review. This caught my attention, that's why I asked the purpose of it. I will do the review in chunks.